### PR TITLE
Shorten cancel timeout for phases to 2 seconds

### PR DESCRIPTION
**Pull request by fahhem**
Also fixes a pickling error that makes updating the measurements' pickle record broken.

### CODE REVIEW

**Comment by Kenadia** [comment]
What's the pickling error and where's the fix?

**Comment by fahhem** [comment]
The fix is the changes in validators, it doesn't use a lambda because you can't pickle those

File: openhtf/util/validators.py [hunk 1]
```diff
@@ -88,7 +90,7 @@ def __init__(self, minimum=None, maximum=None, type=None):
       raise ValueError('Minimum cannot be greater than maximum')
     self.minimum = minimum
     self.maximum = maximum
-    self._type = type or (lambda x: x)
+    self._type = type
```
Comment (grybmadsci) [inline openhtf/util/validators.py]: is there a particular reason not to do `self._type = type or _identity` here?

Comment (fahhem) [inline openhtf/util/validators.py]: pickle is the problem, it wants to pickle the lambda and complains

**Pull request merged**